bsp/esp_bsp_devkit/include/bsp/esp_bsp_devkit.h

```diff
@@ -0,0 +1,292 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief ESP BSP: Generic
+ */
+
+#pragma once
+
+#include "sdkconfig.h"
+#include "driver/gpio.h"
+#include "driver/i2c.h"
```

Comment:
Should we use driver-ng here? Or do you want to add it later?

Reply:
I think that we can keep it as it is and add it later. We will have one version with old driver and new version with NG driver. I think, it can be helpful for this special BSP.